extension.json

```diff
@@ -83,7 +83,14 @@
       },
       "required": ["instances"],
       "additionalProperties": false
-    }
+    },
+    "transforms": [
+      {
+        "type": "function",
+        "propertyPath": "instances[].onBeforeEventSend",
```

Comment:
Support for array notation on `propertyPath` is the functionality we were waiting for from Launch.

Reply:
Hey @Aaronius, is there a public doc we can link to?

Reply:
Yes, there's this: https://developer.adobelaunch.com/extensions/reference/extension-manifest/#transforming-settings-objects

But comments aren't supported in JSON. Would you like to see a comment in configuration.jsx somewhere?

Reply:
Thank you! Only if you think it's useful.